hazelcast-jet-core/src/main/java/com/hazelcast/jet/aggregate/AggregateOperations.java

```diff
@@ -251,7 +298,14 @@ private AggregateOperations() {
      * Returns an aggregate operation that computes the arithmetic mean of the
      * {@code long} values it obtains by applying {@code getLongValueFn} to
      * each item.
+     * <p>
```

Comment:
Please add:
- code sample
- SeeAlso averagingDouble, averagingInt and when to use them (it's the input value that should be long or it;'s the output value of the aggregation?)